src/realm/query.hpp

```diff
@@ -289,6 +289,11 @@ class Query final {
     int set_threads(unsigned int threadcount);
 #endif
 
+    const ConstTableRef& get_table() const noexcept
```

Comment:
`const ConstTableRef` seems redundant and unnecessary, is this really needed?

Reply:
I want to be able to take a `const Query&` and access the table on it. The other implementation of `get_table()` returns a mutable reference. I'd be happy to only have a const ref/const-qualified getter there if that's what you're after.